bcreg-aca/config/settings.yml

```diff
@@ -1,5 +1,5 @@
 default:
   APPLICATION_URL_INCORP: $APPLICATION_URL/bcreg/incorporation
   APPLICATION_URL_DBA: $APPLICATION_URL/bcreg/dba
-  APPLICATION_URL_ADDRESS: $APPLICATION_URL/bcreg/address
-  TEMPLATE_PATH: ../templates
\ No newline at end of file
+  # APPLICATION_URL_ADDRESS: $APPLICATION_URL/bcreg/address
```

Comment:
Looks like `APPLICATION_URL_ADDRESS` is used by the `business_number.registries.ca` service which we're keeping.